static/js/Misc.jsx

```diff
@@ -108,21 +108,64 @@ InterfaceText.propTypes = {
   className: PropTypes.string
 };
 
-const ContentText = ({text, html, overrideLanguage, defaultToInterfaceOnBilingual=false, bilingualOrder = null}) => {
+const VersionContent = (props) => {
+  const renderedItems = ContentTextFilterByLang({...props});
+
+  const languageToFilter = renderedItems.length === 2 ? 'bilingual' : renderedItems[0][0];
+  function isImage(textChunk) {
+      const pattern = /<img\b[^>]*>/i;
+      const isImageTag = pattern.test(textChunk);
+      return isImageTag;
+  }
+  return renderedItems.map((item) => {
+      if (isImage(item[1])){
+        return(<VersionImage renderedItem={item} languageToFilter={languageToFilter} textImageLoadCallback={props.textImageLoadCallback}/>)
+      }
+      return (<ContentSpan item={item} isDangerouslySetInnerHTML={!!props.html}/>)
+  })
+}
+
+const VersionImage = ({renderedItem, languageToFilter, textImageLoadCallback}) => {
```

Comment:
I would break up the `renderedItem` prop into two props (See my comment on `ContentSpan`)

Reply:
Also, can you add a docstring that explains what this component does?